test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-SecureString.Tests.ps1

```diff
@@ -1,8 +1,9 @@
-Describe "ConvertTo-SecureString" -Tags "CI" {
+	Describe "ConvertTo--SecureString" -Tags "CI" {
 
-    Context "Checking return types of ConvertTo-SecureString" {
+    Context "Checking return types of ConvertTo--SecureString" {
```

Comment:
Seems to have an extra - between ConvertTo and SecureString

Reply:
See the previous comment